pom.xml

```diff
@@ -271,6 +313,213 @@
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-web-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-mongo-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-redis-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-cassandra-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-consul-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-kafka-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-amqp-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-rabbitmq-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-sql-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-pg-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-mysql-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>smallrye-axle-postgres-client</artifactId>
+                <version>${version.smallrye-reactive-utils}</version>
+            </dependency>
```

Comment:
I think these dependencies are in maintenance mode too, right @cescoffier ?

Reply:
Axle yes, and so you not be listed. Quarkus does not depend on them anymore.